src/printapp/test/test-api.py

```diff
@@ -101,18 +101,14 @@ def test_upload_file(self):
             response = app.post('/api/upload', data=form_data)
             self.assertEqual(self._status(response), 400)
 
-    @unittest.skip("Skip test which actually prints.")
-    def test_print(self):
-        """Tests the /api/upload and /api/print endpoints.
-
-        This test is disabled by default. Enable it by commenting out the
-        decorator above.
+    def test_print_and_deletejob(self):
+        """Tests the /api/upload and /api/print and /api/deletejob/<job_id> endpoints.
         """
         with self.get_client() as app:
             self._sign_in(app)
 
             oid = []
-            file_names = ['test.pdf', 'test.txt', 'test.doc', 'test.docx']
```

Comment:
Even with `test.pdf` not included in the files being tested, I'm still having trouble now getting this test to pass.